src/Blocks/CMakeLists.txt

```diff
@@ -30,8 +31,8 @@ target_sources(
 	BlockDoor.h
 	BlockDropSpenser.h
 	BlockEnchantingTable.h
-	BlockEnderchest.h
 	BlockEndPortalFrame.h
+	BlockEnderchest.h
```

Comment:
What alphabet are you using? Are you sorting uppercase before lower case?

Reply:
ask Notepad++...